library/src/main/java/com/sothree/slidinguppanel/SlidingUpPanelLayout.java

```diff
@@ -397,7 +397,7 @@ public void setTouchEnabled(boolean enabled) {
     }
 
     public boolean isTouchEnabled() {
-        return mIsTouchEnabled && mSlideableView != null && mSlideState != PanelState.HIDDEN;
+        return !mIsTouchEnabled || mSlideableView == null || mSlideState == PanelState.HIDDEN;
```

Comment:
Probably you try to fix this. 
<img width="618" alt="image" src="https://user-images.githubusercontent.com/3314607/152602226-5784ad42-a242-4cd7-90b2-ac36cc422e73.png">

If yes an other name should be used